### ISSUE TITLE
support creating a webserver config for apache/nginx

### ISSUE THREAD

**Issue by tosmi** [lesaux/puppet-kibana4#59]
cristifalcas/kibana supports creating an apache vhost or nginx config entry to serve kibana behind a web server, which we would like to use. if it's ok, i'm going to implement this for this module as well and create a PR (i'm going to let myself be inspired by cristifalcas/kibana).

thanks toni

**Comment by lesaux** [comment]
Hi Tosmi, this is useful, but couldn't this be achieved using the apache or nginx puppet modules?

**Comment by tosmi** [comment]
absolutely, i would just add a manifest that adds an apache::vhost entry. e.g. like

https://github.com/cristifalcas/puppet-kibana/blob/master/manifests/proxy/apache.pp

but after sleeping one night over the idea, maybe this could also be implemented in a profile. dunno, if you say yes that's going to be useful, i'll send a PR. if you think, baah not in my module, i'm also ok.

toni
